### DISCUSSION TITLE
Team Updates

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#31705] [General] [answered]
On the volunteer portal there is a section called department and team reports. This is a really useful way for those active contributors who are not on teams to know what is going on, what decisions are made and where help is required. 

However the production department and teams very rarely post any reports.  Even a one line summary of a meeting or decision would be of benefit. 

This is part of the OSM policy for [Department Leaders](https://www.opensourcematters.org/organisation/directors/policies/303-department-coordinator-responsibilities-policy.html) and [Team Leaders](https://www.opensourcematters.org/organisation/directors/policies/296-team-leader-responsibilities-policy.html) so I don't understand why this is not being done.

Also in the past there have been reports of motions that have been passed by the Production Department where there has been subsequent discussion and requests for clarification etc but no further update. The [registry of motions](https://www.opensourcematters.org/organisation/team-membership/registry-of-motions/289-2020.html#production-department-motions-2020) has not been updated since June - have there been no motions.

**Comment by wilsonge** [comment]
This has been raised in the Production Glip (obviously there has been motions (e.g. the RFC stuff)) - should be updated in the coming week or two

**Reply by brianteeman** [reply]
Obvious to you perhaps

**Comment by brianteeman** [comment]
Polite reminder that the registry of motions has still not been updated since June 2000 and that reports are still not being published

**Comment by sozzled** [comment]
It's May 2021 ... still nothing ...

**Comment by sozzled** [comment, marked as answer]
The [registry of motions](https://www.opensourcematters.org/organisation/team-membership/registry-of-motions/308-2021.html) has now been updated, at least as far as the Production Department is concerned.

A shame we can't say the same about [board members reports](https://www.opensourcematters.org/organisation/directors/board-members-reports/307-2021-board-members-quarterly-reports.html)
